4-tools/presentation.ipynb

```diff
@@ -512,66 +473,74 @@
     }
    },
    "source": [
-    "If you're working in a Jupyter notebook, you can use the magic command called [pycodestyle_magic](https://github.com/mattijn/pycodestyle_magic) instead. A demonstration:\n",
+    "# [Black](https://pypi.org/project/black/)\n",
     "\n",
-    "**Note**: using `flake8=>4.0.0` will not let you use the `%flake8_on` magic command"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "scrolled": true,
-    "slideshow": {
-     "slide_type": "-"
-    }
-   },
-   "outputs": [],
-   "source": [
-    "%load_ext pycodestyle_magic\n",
-    "# Using lines longer than 80 characters within a notebook could be reasonable\n",
-    "%flake8_on --max_line_length 99"
+    "Black is an automatic code formater. When Black is run on a script it will change to code into the Black style, taking the decision away from the use. This means that code formatted with Black has a very consistent look.\n",
```

Comment:
```suggestion
    "`black` is an automatic code formater. When `black` is run on a script it will change to code into its style, taking the decision away from the user. This means that code formatted with `black` has a very consistent look.\n",
```